backend/dataall/core/environment/cdk/pivot_role_core_policies/s3.py

```diff
@@ -17,7 +17,8 @@ def get_statements(self):
                 actions=[
                     's3:ListAllMyBuckets',
                     's3:GetBucketLocation',
-                    's3:PutBucketTagging'
+                    's3:PutBucketTagging',
+                    's3:GetEncryptionConfiguration'
```

Comment:
Disclaimer: I thought about restricting this policy and add it to the imported buckets only. The issue is that we check the Bucket encryption before we import it, we check the Bucket encryption before we add pivot role permissions to the Bucket. Since it is a read permission that only reads configuration and not data it is acceptable to be added to the pivot role